index.html

```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
                 <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
             </ul>
             </div>
-
 </section>
 </section>
+<section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (1).png" alt=""></a>
```

Comment:
- [ ] Kindly consider using this picture through CSS as a background image as decorations are always backgrounded images, not img tags.